clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/EnvIntegrationSpec.scala

```diff
@@ -34,12 +34,16 @@ abstract class EnvIntegrationSpec(env: String)
     s"http://elasticsearch1.gotc-$env.broadinstitute.org:9200"
   )
 
-  override lazy implicit val bearerToken: OAuth2BearerToken =
-    OAuth2BearerToken(
-      AuthUtil
-        .getCredsFromServiceAccount(serviceAccount)
-        .getTokenValue
+  override lazy implicit val bearerToken: OAuth2BearerToken = {
+    val tokenOrErr = AuthUtil
+      .getCredsFromServiceAccount(serviceAccount)
+      .map(_.getTokenValue)
+
+    tokenOrErr.fold(
+      fail(s"Couldn't get access token for account $serviceAccount", _),
+      OAuth2BearerToken.apply
```

Comment:
I don't understand the constructor is called here instead of in `AuthUtil`. It looks like every use of `AccessToken` outside of `AuthUtil` is only used to create an `OAuth2BearerToken`. Wouldn't it be simpler to have `AuthUtil` return the bearer token directly?

Reply:
It would be simpler. @tlangs had it set up that way originally but I asked him to change it because we have to add all of akka-http as a dependency to clio-util to make it happen, and I was hoping to keep that project's dependencies slim (though maybe we don't care, since we're building fat jars everywhere anyways).